e2e/fixtures/events/createClaimSpec.js

```diff
@@ -36,6 +36,24 @@ const claimAmount = '150000';
 
 const validPba = listElement('PBAFUNC12345');
 const invalidPba = listElement('PBA0078095');
+const value={
+  value:{
+    code: 'OTHER',
+    label: 'OTHER'
+  }
+};
+
+const airLineList={
+  ...value
+};
+const flightDelayDetails ={
+  airlineList:{
+    ...airLineList,
+  },
+  flightNumber: 'test',
```

Comment:
I would put more realistic values here, for example:
flightNumber: 'SK247',
 nameOfAirline: 'Sky Airways',